wiki/Legal/Terms/id.md

```diff
@@ -0,0 +1,118 @@
+osu! tidak mengizinkan aktivitas yang melanggar hak cipta dan pelanggaran hak kekayaan intelektual di Situs Web osu!, dan osu! akan menghapus semua Konten dan Kiriman Pengguna jika diberitahukan dengan benar bahwa Konten atau Kiriman Pengguna tersebut melanggar hak kekayaan intelektual milik orang lain.
+osu! berhak untuk menghapus Konten dan Kiriman Pengguna tanpa pemberitahuan sebelumnya.
+
+## TANGGUNG JAWAB ANDA SEBAGAI SEORANG RELAWAN
+
+osu! berisi sejumlah tim sukarelawan pada waktu tertentu yang diambil langsung dari komunitas. Tim tertentu memerlukan izin dan akses ekstra untuk memfasilitasi aspek pada komunitas dari gim. Tim tersebut adalah sebagai berikut:
+
+- [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team)
+- [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team)
+
+Tidak ada pengguna di bawah usia 18 tahun yang dapat menjadi anggota dari salah satu tim yang tercantum di atas.
+
+## KEBIJAKAN PRIVASI
+
+osu! menganggap privasi Anda adalah yang paling penting, dan kami menghargai penggunaan Anda terhadap layanan kami.
+Layanan osu! tidak dipungut biaya apa pun, tidak diperlukan pendaftaran sebelum menggunakan layanan ini.
+osu! akan secara eksplisit meminta informasi tentang Anda ketika informasi tersebut diperlukan untuk penggunaan layanan yang disediakan oleh osu!.
+Setiap pengiriman informasi pribadi dilakukan berdasarkan keinginan Anda sendiri (situs web), atau secara otomatis (perangkat lunak klien gim) jika diperlukan untuk memberikan diagnosis dan umpan balik.
+osu! merahasiakan semua informasi pribadi, dan tidak akan membagikan informasi ini kepada pihak ketiga mana pun kecuali dalam keadaan berikut:
+
+Ketika tindakan tersebut diperlukan untuk memenuhi persyaratan hukum atau mematuhi proses hukum.
+
+Ketika tindakan tersebut diperlukan untuk melindungi dan mempertahankan hak atau properti osu!.
+
+Ketika tindakan tersebut diperlukan untuk menegakkan syarat dan ketentuan dari layanan osu!.
+
+osu! dapat menggunakan cookie sebagai alat dalam proses pelacakan, tetapi cookie tidak akan digunakan untuk mengidentifikasi Anda secara pribadi tanpa persetujuan Anda sebelumnya.
+Jika pengguna memutuskan untuk berpartisipasi di forum, dengan persetujuan dari pengguna, sebuah cookie akan dikeluarkan untuk melacak nama pengguna dan kata sandi dalam periode 1 tahun.
+
+## HAK KEPEMILIKAN ATAS KONTEN
+
+Pengguna mengakui bahwa konten, termasuk teks, perangkat lunak, musik, suara, foto, video, grafik, referensi nama atau materi lain yang terkandung dalam hasil pencarian iklan sponsor, yang disediakan atau diakses oleh Pengguna melalui Layanan ("Konten") oleh osu!, dilindungi oleh hak cipta, merek dagang, merek layanan, paten, atau hak milik dan undang-undang lainnya; oleh karena itu, Pengguna diperbolehkan untuk menggunakan Konten ini sebagaimana yang telah diizinkan secara tersurat oleh penyedia Layanan.
```

Comment:
```suggestion
Pengguna mengakui bahwa konten, termasuk teks, perangkat lunak, musik, suara, foto, video, grafik, referensi nama, atau materi lain yang terkandung dalam hasil pencarian iklan sponsor yang disediakan atau diakses oleh Pengguna melalui Layanan ("Konten") oleh osu! dilindungi oleh hak cipta, merek dagang, merek layanan, paten, atau hak milik dan undang-undang lainnya. Oleh karena itu, Pengguna diperbolehkan untuk menggunakan Konten ini sebagaimana yang telah diizinkan secara tersurat oleh penyedia Layanan.
```
fixed some commas and made a new sentence because "Oleh karena itu" is a intersentence conjunction